packages/apps/transfer/src/components/Global/Ace/index.tsx

```diff
@@ -8,28 +8,43 @@ import 'ace-builds/src-noconflict/theme-monokai';
 import 'ace-builds/src-noconflict/theme-one_dark';
 import 'ace-builds/src-noconflict/ext-language_tools';
 
+export interface IOnchange {
+  (value: string): void;
+}
+
 export interface IAceEditorProps {
   code?: string;
+  width?: string;
+  height?: string;
+  readonly?: boolean;
+  onChange?: IOnchange;
 }
 
-const AceViewerComponent: FC<IAceEditorProps> = ({ code }) => (
+const AceViewerComponent: FC<IAceEditorProps> = ({
+  code,
+  width,
+  height,
+  readonly,
+  onChange,
+}) => (
```

Comment:
I know Ace Editor has support for Vim-mode. Can we enable that with a toggle? Not blocking for this PR, just a thought